src/components/Definition/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from "react";
+import * as styles from "./Definition.module.scss";
+
+export default function Definition({ word }) {
+  const [definition, setDefinition] = useState("");
+
+  useEffect(() => {
+    if (word) {
+      async function fetchData() {
+        const result = await fetch(
+          `https://www.dictionaryapi.com/api/v3/references/sd2/json/${word}?key=${process.env.GATSBY_MERRIAM_APIKEY}`
+        ).then((res) => res.json());
+        setDefinition(
+          result[0].shortdef[0].replace(/^\w/, (firstLetter) =>
+            firstLetter.toUpperCase()
+          )
+        );
+      }
+      fetchData();
+    }
+  }, [word]);
+
+  return (
+    <div className={styles.definition_container}>
```

Comment:
We might want to add a conditional render here that if there is no data, nothing displays (in case the API goes down or something) that way there isn't a div that just says "Definition ."

Reply:
Good idea!